guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -31,6 +31,9 @@ endif::[]
 
 Log files are written to `/var/log/messages/,` `/var/log/httpd/`, and `/var/lib/foreman-proxy/openscap/content/`.
 You can manage the size of these files using *logrotate*.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/solutions/1294[logrotate utility solution].
```

Comment:
```suggestion
For more information, see https://access.redhat.com/solutions/1294[How to use logrotate utility to rotate log files].
```
Pls, use the exact title of the article :)

Reply:
Addressed :+1: